lib/screens_web/views/v2/audio/elevator_status_view.ex

```diff
@@ -9,16 +9,24 @@ defmodule ScreensWeb.V2.Audio.ElevatorStatusView do
       }) do
     ~E|
     <p>Elevator Closures across the T.</p>
-    <p><%= render_cta() %></p>
     <p><%= render_active_at_home(active_at_home_pages) %></p>
     <p>Other elevator closures:</p>
-    <p><%= Enum.map(list_pages, &render_page/1) %></p>
-    <p><%= Enum.map(upcoming_at_home_pages, &render_page/1) %></p>
-    <p><%= Enum.map(elsewhere_pages, &render_page/1) %></p>
-    <p><%= render_cta() %></p>
+    <p><%= render_closures(list_pages, upcoming_at_home_pages, elsewhere_pages) %></p>
     |
   end
 
+  defp render_closures(list_pages, upcoming_at_home_pages, elsewhere_pages) do
+    if Enum.empty?(list_pages) and Enum.empty?(upcoming_at_home_pages) and
+         Enum.empty?(elsewhere_pages) do
```

Comment:
This isn't really necessary, more of a nit and practice with Elixir, but you could use pattern matching instead of having all these if statements. 
```
defp render_closures([], [], []) do
  "All other MBTA elevators are working or have a backup elevator within 20 feet."
end
```

Then the next function matching could be defined as `defp render_closures(list_pages, upcoming_at_home_pages, elsewhere_pages)` and handle the other logic